test/unit/spec/LoginRouter_spec.js

```diff
@@ -170,21 +170,9 @@ function (Okta, Q, Backbone, SharedUtil, CryptoUtil, Logger, OktaAuth, Util, Exp
       });
     }
 
-    function expectUnexpectedFieldLog(arg1) {
-      expect(Logger.warn).toHaveBeenCalledWith('Field not defined in schema', arg1);
-    }
-
-    it('throws a ConfigError if unknown option is passed as a widget param', function () {
```

Comment:
The description of this test (and below a bit) do not match the assertions.

Since we do not inject additional params anywhere, we should be OK **not** throwing an error here. As an alternative, we *could* add conditional logic [here](https://github.com/okta/okta-signin-widget/blob/master/src/widget/OktaSignIn.js#L87).